frontend/app/views/fragments/bundle/bundleOfferingsB.scala.html

```diff
@@ -1,3 +1,196 @@
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>Saturday</h1>
+                            <p></p>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £23.40/month
+                    </div>
+                    </li>
+                    <li class="bundle-offering-b__subscribe__offer__print-options__option">
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>The Guardian Weekly</h1>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £23.40/month
+                    </div>
+                    </li>
+                    <li class="bundle-offering-b__subscribe__offer__print-options__option">
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>Weekend</h1>
+                            <p>Saturday & Sunday</p>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £26.00/month
+                    </div>
+                    </li>
+                    <li class="bundle-offering-b__subscribe__offer__print-options__option">
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>Saturday & The&nbsp;Guardian Weekly</h1>
+                            <p></p>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £28.60/month
+                    </div>
+                    </li>
+                    <li class="bundle-offering-b__subscribe__offer__print-options__option">
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>Sixday</h1>
+                            <p>Monday - Saturday</p>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £88.88/month
+                    </div>
+                    </li>
+                    <li class="bundle-offering-b__subscribe__offer__print-options__option">
+                        <div class="bundle-offering-b__print-option__description">
+                            <h1>Everyday</h1>
+                            <p>Monday - Sunday</p>
+                        </div><div class="bundle-offering-b__print-option__price">
+                                    £99.99/month
+                    </div>
+                    </li>
+                </ul>
+                @fragments.bundle.bundleButton("£23.40/month","#","")
+            </div>
+        </div>
+        </div>
+    </div>
+    <div class="bundle-offering-b__give">
+        <div class="bundle-offering-b__line"></div>
+        <div class="bundle-offering-b__give__content">
```

Comment:
Seems like there are few unnecessary elements here? The hierarchy could become a bit neater and more semantic:

```
<div class="bundle-offering-b__give">
  <hr class="bundle-offering-b__line">
  <h1 class="bundle-offering-b__give__heading">Choose to support ... </h1>
  <p class="bundle-offering-b__give__description">Fund our journalism ... </p>
  @fragments.bundle.bundleButton("Contribute now","#","", secondary = true)
</div>
```